provision/dev/nginx_site_docker.conf

```diff
@@ -0,0 +1,11 @@
+server {
+  listen 80 default_server;
+  server_name _;
+
+  location / {
+  proxy_set_header X-Forwarded-Email "aaron.snow@gsa.gov";
+    proxy_set_header Host 192.168.33.10;
+  proxy_set_header X-forwarded-for $proxy_add_x_forwarded_for;
+    proxy_pass http://app:8000;
```

Comment:
I'm not a big fan of this because it's just a copy of `nginx_site.conf`.

Ideally we could use [environment variables](https://github.com/docker-library/docs/tree/master/nginx#using-environment-variables-in-nginx-configuration) here, but this would require changing the vagrant setup too, and I'm unfamiliar with vagrant.

Another option would be to do some kind of janky replacement of `http://127.0.0.1` with `http://app` from `nginx_site.conf`, but that feels janky, and makes any docker-specific customizations we might want to do in the future that much harder.

It might just be easiest to violate DRY a bit here, which shouldn't be *that* bad, especially since this dev nginx configuration isn't likely to change that often. The two configs are also in the same directory and named similarly, so it shouldn't be hard to modify both at the same time (especially if we add comments to the files mentioning this).